### PR TITLE
Update console tests to use new NET Core library and use NUnit

### PR DESCRIPTION
**Pull request by jcoliz**
Changes
* Replaced hand-built test runner with NUnit
* Upgraded to .NET Framework 4.8
* Use SDK-style project

Note that I did not change any code or data for the tests themselves. This is purely an infrastructure and dependencies change.

### CODE REVIEW

File: tests/ExcelFinancialFunctions.ConsoleTests/MatrixTests.fs [hunk 1]
```diff
@@ -0,0 +1,133 @@
+﻿namespace Excel.FinancialFunctions
```
Comment (dsyme) [inline tests/ExcelFinancialFunctions.ConsoleTests/MatrixTests.fs:1]: Were some tests deleted?  I just see additions, which is cool, just wondering

Comment (jcoliz) [reply tests/ExcelFinancialFunctions.ConsoleTests/MatrixTests.fs:1]: No tests removed or deleted or even modified. At this point I probably don't have the F# skills to be making changes like that 😃

Just reorganized HOW they were run. I did remove the file "testsexec.fs" from the project. Left the file itself in the repository for you to refer to if you wanted. All the work from testsexec.fs has been moved into SpotTests.fs and MatrixTests.fs.

**Review by dsyme — APPROVED** [review]

File: tests/ExcelFinancialFunctions.ConsoleTests/MatrixTests.fs (hunk 1 above)
Comment (dsyme) [reply tests/ExcelFinancialFunctions.ConsoleTests/MatrixTests.fs:1]: I see, yes just remove testexec.fs then?

Comment (jcoliz) [reply tests/ExcelFinancialFunctions.ConsoleTests/MatrixTests.fs:1]: Will do. Was going to come back and do that as a cleanup

**Pull request merged**